README.Mac.md

```diff
@@ -1,294 +1,289 @@
-
+**a.** Find & modify the `ServerName` to:
 ```
-sudo mv /path/to/Smarty-3.x.x/libs/ /usr/local/lib/php/Smarty/smarty3
+ServerName localhost
 ```
 
-After you have moved the directory ensure to add the path ```/usr/local/lib/php/Smarty``` to the ```include_path``` in the php.ini file (see [section 1.3.4](#include_path) part 3).
-
-# 2. Installing LORIS Code Base
+**b.** Find the lines for `DocumentRoot` and `Directory` in Apache and change them to:
+```
+DocumentRoot "/Users/<your_user>/Development/GitHub/Loris/htdocs/"
+<Directory "/Users/<your_user>/Development/GitHub/Loris/htdocs/">
+```
 
-This section goes over how to obtain the LORIS code base and setup the database on your local machine.
+**c.** In the same `<Directory>` block, modify `AllowOverride` to allow all:
+```
+# AllowOverride controls what directives may be placed in .htaccess files.
+# It can be "All", "None", or any combination of the keywords:
+#   AllowOverride FileInfo AuthConfig Limit
+#
+AllowOverride All
+```
 
-## 2.1 Obtaining the LORIS code
+**d.** Enable `mod_rewrite` which is commented out by default:
+```
+LoadModule rewrite_module lib/httpd/modules/mod_rewrite.so
+```
 
-Download the latest release from the [releases page](https://github.com/aces/Loris/releases) and
-extract it to `~/Sites/$projectname`
+**e.** Modify the `User` & `Group` (replace your_user again with your macOS user) to:
+```
+User your_user
+Group staff
+```
 
-<i>$projectname ⇾ “loris” or one-word project name</i>
+#### Restart Apache to load the updated settings
+```
+sudo apachectl -k restart
+```
 
-## 2.2 Installing LORIS
+## 1.6 Install PHP and pkg-config
 
-Run installer script to install core code, libraries, and MySQL schema (see Loris Installation Schematic). 
-The script will prompt for information, including usernames and folders which it will create automatically.
+Paste these commands into your terminal and hit enter:
 ```
-cd ~/Sites/$projectname/tools
-./install.sh
+brew install php72
+brew link php72
+brew services start php72
+brew install pkg-config
 ```
 
-The install script may not fully execute on Mac OS systems.  It is recommended to carefully review and execute manually the steps in the install.sh script on your system.  A few MySQL statement examples are provided in section 2.3. 
-Since the install script was created for LINUX OS, its apache2 setup steps won’t work. Section 3 describes how to set up the apache2 server for Mac.
+### 1.6.1 Modify Apache Configuration for PHP
 
-## 2.3 Manually running install script steps
+Modify the Apache configuration file to enable php.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
+```
 
-Since you may not be able to fully execute the install.sh script at step 2.2, the following examples may help in manually executing the sequence of commands found in the install script.
+**a.** Add the below line to `LoadModules` (Change `php@<version>` to the correct version):
```

Comment:
I would say 'the line below' instead of the 'the below line'. No?

Reply:
@cmadjar Yah that's better. Fixed.